tests/tasks/sql_server/__init__.py

```diff
@@ -1,3 +1,4 @@
 import pytest
 
-pytest.importorskip("pyodbc")
+pytest.skip("CI required extra SQL dependencies")
```

Comment:
Can this be an `importorskip` ?

Reply:
I removed the logic I put there, forgot we handle for this in the installs for the CI job